butterfly-utilities/src/main/java/com/paypal/butterfly/utilities/file/LocateFile.java

```diff
@@ -23,6 +23,7 @@
  */
 public class LocateFile extends TransformationUtility<LocateFile> {
 
+    private static final String DESCRIPTION_RELATIVE_MISSING = "Locate file cannot be determined";
```

Comment:
@amriteya please rephrase this to `Locate specified file`. This sounds more gentle I believe.